CHANGES.rst

```diff
@@ -258,6 +258,9 @@ Bug Fixes
 
   - Handle sorting NaNs and masked values in jsviewer. [#4052, #5572]
 
+  - Ensure mixin columns can be set to scalars if the type supports
```

Comment:
And of course it is actually a `scalar or length-1 array value`, but maybe that's too fine a distinction here?